src/Error.mo

```diff
@@ -17,6 +17,8 @@ module {
   ///   #system_fatal;
   ///   // Transient error.
   ///   #system_transient;
+  ///   // Unknown execution status due to expired deadline.
```

Comment:
is `missed deadline` better?

Reply:
I would stick to what's in the spec, assuming that's what we did with the others (I don't recall)

Reply:
Good point, will have a look!

Reply:
https://github.com/dfinity/portal/pull/3764/files#diff-1c97553a3ab60d2141abdafed839771c625f6d7f281398197929a8a9d073af8fR1116
But that is a bit verbose. I'll go with
```suggestion
  ///   // Response unknown due to missed deadline.
```

Reply:
I would stick to the original wording, just for consistency.

Adding this case is actually a breaking change. Maybe that's ok, but the alternative is to not change it and add a predicate that tests for #future 6 (or whatever the code is). I guess the advantage of the breaking change is that people will need to adjust their code.